src/sidebar/helpers/test/view-filter-test.js

```diff
@@ -187,6 +193,12 @@ describe('sidebar/helpers/view-filter', () => {
       assert.deepEqual(result, [anns[1].id, anns[2].id, anns[3].id]);
     });
 
+    it('matches username and display name independently', () => {
+      const anns = [annotationWithUser('dean31', 'James Dean')];
+      const result = filterAnnotations(anns, userQuery('dean31 james'));
+      assert.equal(result.length, 0);
+    });
```

Comment:
Previously the username and display name were concatenated into a single string (`${username} ${displayName}`) which the query term was matched against. This results in a very minor bug where queries that shouldn't match do. The query is now tested separately against the display name and username.